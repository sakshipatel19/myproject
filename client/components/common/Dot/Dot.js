import React from "react";

import "./Dot.scss";
const Dot = ({ color }) => {
	return <span className="dot" style={{ backgroundColor: color }}></span>;
};

export default Dot;
