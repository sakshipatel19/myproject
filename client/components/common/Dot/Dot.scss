.dot {
	height: 8px;
	width: 8px;
	background-color: #bbb;
	border-radius: 50%;
	display: inline-block;
	margin-right: 5px;
	margin-top: 10px;
}
