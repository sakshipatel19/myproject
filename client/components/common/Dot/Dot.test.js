import React from "react";
import { shallow } from "enzyme";

import Dot from "./Dot";

describe("Dot tests", () => {
	it("Dot should render correctly", () => {
		const wrapper = shallow(<Dot color="#fff" />);
		expect(wrapper).toMatchSnapshot();
	});
});
