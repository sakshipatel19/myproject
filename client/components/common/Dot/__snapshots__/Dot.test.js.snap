// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dot tests Dot should render correctly 1`] = `
<span
  className="dot"
  style={
    Object {
      "backgroundColor": "#fff",
    }
  }
/>
`;
